platform/macos/docs/guides/Migrating to Expressions.md

```diff
@@ -5,7 +5,7 @@
 
 # Migrating from Style Functions to Expressions
 
-[Runtime Styling](runtime-styling.html) enables you to modify every aspect of the map’s appearance dynamically as a user interacts with your application. Developers can specify in advance how a layout or paint attribute will vary as the zoom level changes or how the appearance of individual features vary based on metadata provided by a content source.
+[Runtime Styling](https://www.mapbox.com/ios-sdk/maps/overview/runtime-styling/) enables you to modify every aspect of the map’s appearance dynamically as a user interacts with your application. Developers can specify in advance how a layout or paint attribute will vary as the zoom level changes or how the appearance of individual features vary based on metadata provided by a content source.
```

Comment:
Kinda awkward that this macOS appendix is linking to an explicitly iOS-specific guide, but it’s better than nothing I guess.